src/server/scripts/Northrend/IcecrownCitadel/icecrown_citadel.cpp

```diff
@@ -15,290 +15,159 @@
+    SAY_SAURFANG_INTRO_1 = 15,
+    SAY_SAURFANG_INTRO_2 = 16,
+    SAY_SAURFANG_INTRO_3 = 17,
+    SAY_SAURFANG_INTRO_4 = 18,
 
     // Muradin Bronzebeard (at Light's Hammer)
-    SAY_MURADIN_INTRO_1             = 13,
-    SAY_MURADIN_INTRO_2             = 14,
-    SAY_MURADIN_INTRO_3             = 15,
+    SAY_MURADIN_INTRO_1 = 13,
+    SAY_MURADIN_INTRO_2 = 14,
+    SAY_MURADIN_INTRO_3 = 15,
 
     // Deathbound Ward
-    SAY_TRAP_ACTIVATE               = 0,
+    SAY_TRAP_ACTIVATE = 0,
 
     // Rotting Frost Giant
-    EMOTE_DEATH_PLAGUE_WARNING      = 0,
-
-    // Sister Svalna
-    SAY_SVALNA_KILL_CAPTAIN         = 1, // happens when she kills a captain
-    SAY_SVALNA_KILL                 = 4,
-    SAY_SVALNA_CAPTAIN_DEATH        = 5, // happens when a captain resurrected by her dies
-    SAY_SVALNA_DEATH                = 6,
-    EMOTE_SVALNA_IMPALE             = 7,
-    EMOTE_SVALNA_BROKEN_SHIELD      = 8,
-
-    SAY_CROK_INTRO_1                = 0, // Ready your arms, my Argent Brothers. The Vrykul will protect the Frost Queen with their lives.
-    SAY_ARNATH_INTRO_2              = 5, // Even dying here beats spending another day collecting reagents for that madman, Finklestein.
-    SAY_CROK_INTRO_3                = 1, // Enough idle banter! Our champions have arrived - support them as we push our way through the hall!
-    SAY_SVALNA_EVENT_START          = 0, // You may have once fought beside me, Crok, but now you are nothing more than a traitor. Come, your second death approaches!
-    SAY_CROK_COMBAT_WP_0            = 2, // Draw them back to us, and we'll assist you.
-    SAY_CROK_COMBAT_WP_1            = 3, // Quickly, push on!
-    SAY_CROK_FINAL_WP               = 4, // Her reinforcements will arrive shortly, we must bring her down quickly!
-    SAY_SVALNA_RESURRECT_CAPTAINS   = 2, // Foolish Crok. You brought my reinforcements with you. Arise, Argent Champions, and serve the Lich King in death!
-    SAY_CROK_COMBAT_SVALNA          = 5, // I'll draw her attacks. Return our brothers to their graves, then help me bring her down!
-    SAY_SVALNA_AGGRO                = 3, // Come, Scourgebane. I'll show the master which of us is truly worthy of the title of "Champion"!
-    SAY_CAPTAIN_DEATH               = 0,
-    SAY_CAPTAIN_RESURRECTED         = 1,
-    SAY_CAPTAIN_KILL                = 2,
-    SAY_CAPTAIN_SECOND_DEATH        = 3,
-    SAY_CAPTAIN_SURVIVE_TALK        = 4,
-    SAY_CROK_WEAKENING_GAUNTLET     = 6,
-    SAY_CROK_WEAKENING_SVALNA       = 7,
-    SAY_CROK_DEATH                  = 8,
+    EMOTE_DEATH_PLAGUE_WARNING = 0,
 };
 
 enum ICCSpells
 {
     // Rotting Frost Giant
-    SPELL_DEATH_PLAGUE              = 72879,
-    SPELL_DEATH_PLAGUE_AURA         = 72865,
-    SPELL_RECENTLY_INFECTED         = 72884,
-    SPELL_DEATH_PLAGUE_KILL         = 72867,
-    SPELL_STOMP                     = 64652,
-    SPELL_ARCTIC_BREATH             = 72848,
+    SPELL_DEATH_PLAGUE = 72879,
```

Comment:
remove enum identation for?

Reply:
codestyle consistency, all of them where fcked up, but I can add em back correctly :)